docs/spec/0.95.0/4_Baseline_Parameters.md

```diff
@@ -0,0 +1,173 @@
+# Baseline Parameters
+_ElectionGuard_ uses integer ElGamal rather than elliptic-curve ElGamal in order to make construction of election verifiers as simple as possible without requiring special tools and dependencies. The exponential ElGamal used to encrypt votes is defined by a 4096-bit prime $p$ and a 256-bit prime $q$ which divides $(p − 1)$. We use $r = \frac{p-1}{q}$ to denote cofactor of $q$, and a generator $g$ of the order $q$ subgroup $\mathbb{Z}^{r}_{p}$ is fixed.
+
+Another parameter of an election should be a public ballot coding file. This file should list all the contests in an election, the number of selections allowed for each contest, and the options for each contest together with associations between each option and its representation on a virtual ballot. It is assumed that each contest in the ballot coding file has a unique label and that within each contest, each option also has a unique label. For instance, if Alice, Bob, and Carol are running for governor, and David and Ellen are running for senator, the ballot coding file could enable the vector$〈0,1,0; 0,1〉$ to be recognized as a ballot with votes for Bob as governor and Ellen as senator. The detailed format of a ballot coding file will not be specified in this document. But the contents of this file are hashed together with the prime modulus $(p)$, subgroup order $(q)$, generator $(g)$, number of guardians $(n)$, decryption quorum threshold value $(k)$, date, and jurisdictional information to form a base hash code $(Q)$ which will be incorporated into every subsequent hash computation in the election.
+
+Standard parameters for ElectionGuard begin with the largest 256-bit prime $q = 2^{256} − 189$. The hexadecimal representation of q is as follows.
+
+```
+FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFF43
+```
+The modulus $p$ is then set to be a 4096-bit prime with the following properties.
+
+- The first 256 bits of $p$ are all ones.
+- The last 256 bits of $p$ are all ones.
+- $p − 1$ is a multiple of $q$.
+- $\frac{p−1}{2q}$ is also prime.
+
+The middle 3584 bits of $p$ are chosen by starting with the first 3584 bits of the Euler–Mascheroni constant ($\gamma$), pre-pending and appending 256 ones, and finding the smallest prime larger than this value that satisfies the above properties.
+
+This works out to $p = 2^{4096} − 2^{3840} − 1 + 2^{256}(⌊2^{3584}\gamma⌋ + \delta)$ where the value of $\delta$ is given by
+
+$$
+\delta = 495448529856135475846147600290107731951815687842437876083937612367400355133042233301 .^{12}
```

Comment:
This is missing a [footnote](https://squidfunk.github.io/mkdocs-material/reference/footnotes/) 12. This should the footnote notation so that it is linked and includes the text. 

Footnote 12 should be relabeled to 2 since it is the second footnote of this document.